scripts/build-sunshine.sh

```diff
@@ -26,13 +27,14 @@ absolute_path() {
 
 CMAKE_BUILD_TYPE="-e CMAKE_BUILD_TYPE=Release"
 SUNSHINE_PACKAGE_BUILD=OFF
+SUNSHINE_PACKAGE_EXTENSION=deb
 SUNSHINE_GIT_URL=https://github.com/sunshinestream/sunshine.git
 CONTAINER_NAME=sunshine
 
 # Docker will fail if ctrl+c is passed through and the input is not a tty
 DOCKER_INTERACTIVE=-ti
 
-while getopts ":dpuhc:s:n:" arg; do
+while getopts ":dpeuhc:s:n:" arg; do
```

Comment:
This is the problem. 99% sure!

`while getopts ":dpuhc:e:s:n:" arg; do`

The first values (dpuhc) don't have values attached.

Reply:
I guess I didn't change that. Thought I did.